src/browser/graphics/components/lib/fit-text.tsx

```diff
@@ -45,9 +46,13 @@ export const FitText: FunctionComponent<{
 				...props.style,
```

Comment:
FitTextは基本的に中央揃えになるように表示されるので、明示的にtext-align: centerを付与しちゃうとか

```suggestion
				textAlign: "center",
				lineHeight: `${size + 4}px`,
				...props.style,
```

<img width="764" alt="image" src="https://user-images.githubusercontent.com/3125070/234039494-f45f1d10-a74c-44e2-a413-d3f8d3b339ed.png">

Reply:
FitTextの中身はgridでplace-items: centerなので`<br>`なしでもspanかなにか並べたら中央揃えで並ぶなあとか思ってました